src/main/java/eu/tornplayground/tornapi/models/faction/basic/member/PlayerState.java

```diff
@@ -0,0 +1,33 @@
+package eu.tornplayground.tornapi.models.faction.basic.member;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.io.Serializable;
+
+public enum PlayerState implements Serializable {
```

Comment:
Duplicate of `eu.tornplayground.tornapi.models.user.partial.Status.State`, should probably become a shared class.

Reply:
also unused

Reply:
moved some common classes into a package and fixed classes and tests